test/cluster/mzcompose.py

```diff
@@ -2071,9 +2125,9 @@ def workflow_test_drop_during_reconciliation(c: Composition) -> None:
         c.up("materialized", "clusterd1", "toxiproxy")
         c.up("testdrive", persistent=True)
 
-        # Set up toxi-proxies for all clusterd endpoints.
+        # Set up toxi-proxies for clusterd GRPC endpoints.
```

Comment:
Excluding the Timely runtime endpoints from getting toxiproxied. This change is not strictly necessary here but already prepares for the removal of the `COMPUTE ADDRESSES`/`STORAGE ADDRESSES` options.